keyboards/crkbd/keymaps/olligranlund_nordic/keymap.c

```diff
@@ -0,0 +1,131 @@
+#include QMK_KEYBOARD_H
+#include "keymap_swedish.h"
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
```

Comment:
Just a heads up.... it's come up recently that having this set to 16 can increase your firmware size, just by having this at a high number. 

Ideally, this should be lower, especially if you're having issues with compiled firmware size. 

If that's not an issue here, then you can ignore this.

Reply:
It hasn't been an issue, but apparently something really got messed up since this keymap shoulda been under "iris" and now under "crkbd".... I really need to re-check this